vspc/server.py

```diff
@@ -195,6 +198,24 @@ async def save_to_log(self, uuid, data):
         fpath = os.path.join(CONF.serial_log_dir, uuid)
         async with aiofiles.open(fpath, 'ab') as f:
             await f.write(data)
+        if CONF.log_file_size and os.path.getsize(fpath) > CONF.log_file_size:
+            truncated_data = await self.truncate_log_file(fpath, CONF.log_file_size)
+            await self.rewrite_file(fpath, truncated_data)
+
+    async def truncate_log_file(self, file_name, size):
```

Comment:
I find the name confusing. It doesn't actually truncate the log file, but just returns the truncated data. From the name it sounded like it would also change the file. Can we find a better name or move the `rewrite_file()` into the function so it actually changes the file? I don't see it used anywhere else.

Reply:
Sure, sounds like it is better to move `rewrite_file` to `truncate_log_file`